lib/seek/bio_schema/resource_decorators/base_decorator.rb

```diff
@@ -26,7 +26,7 @@ def attributes
 
         # The @context to be used for the JSON-LD
         def context
-          'http://schema.org'
+          'https://schema.org'
```

Comment:
Not sure about this...

The schema.org site uses both http and https in examples, but the RO-Crate context uses http. Maybe @stain can comment

Reply:
I was unsure about this too. https didn't used to validate using the structured data tool, and the schema.org examples are quite mixed up. I double checked and https now does seem to validate

Reply:
Yes. The choice confused me too. The advice from schema.org doesn't really help

There is a general trend towards using 'https' more widely, and you can already write 'https://schema.org' in your structured data. Over time we will migrate the schema.org site itself towards using https: as the default version of the site and our preferred form in examples. However 'http://schema.org' -based URLs in structured data markup will remain widely understood for the forseeable future and there should be no urgency about migrating existing data. This is a lengthy way of saying that both 'https://schema.org' and 'http://schema.org' are fine

Reply:
perhaps we just need to be consistent with bioschemas.org and ro-crate and migrate together

Reply:
http://schema.org is forwarding to https://schema.org so that is the one we should use